pkgs/build-support/fetchcipd/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenvNoCC, writeText, cipd, cacert }:
+with lib;
+{ name ? "source", package, version, sha256 ? fakeHash }:
+stdenvNoCC.mkDerivation {
+  pname = name;
+  inherit version package;
+  builder = ./builder.sh;
+
+  SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
or add cacert to nativeBuildInputs

Reply:
Other fetchers don't do that, we only need to set `SSL_CERT_FILE`.